### PR TITLE
Adding support for IBE60

### PR DESCRIPTION
**Pull request by 4pplet**
## Description

Fixed layout (true HHKB) solder PCB for the IBE60 Keyboard

## Types of Changes
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist
- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/4pplet/ibe60/readme.md
Comment (zvecr) [inline keyboards/4pplet/ibe60/readme.md:1]: This file needs to follow the template <https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md>

File: keyboards/4pplet/ibe60/keymaps/via/rules.mk [hunk 1]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
```
Comment (zvecr) [inline keyboards/4pplet/ibe60/keymaps/via/rules.mk:2]: Has no configured encoders
```suggestion
VIA_ENABLE = yes
```

File: keyboards/4pplet/ibe60/ibe60.h
Comment (zvecr) [inline keyboards/4pplet/ibe60/ibe60.h:1]: Please remove this file

File: keyboards/4pplet/ibe60/ibe60.c
Comment (zvecr) [inline keyboards/4pplet/ibe60/ibe60.c:1]: Please remove this file

File: keyboards/4pplet/ibe60/config.h
Comment (zvecr) [inline keyboards/4pplet/ibe60/config.h:1]: Please remove this file

File: keyboards/4pplet/ibe60/keyboard.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    },
+    "processor": "STM32F072",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0001",
+        "vid": "0x4448"
+    },
+    "indicators": {
+        "caps_lock": "A8",
+        "on_state": 1
```
Comment (zvecr) [inline keyboards/4pplet/ibe60/keyboard.json:28]: Duplicates defaults
```suggestion
        "caps_lock": "A8"
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/4pplet/ibe60/readme.md
Comment (4pplet) [reply keyboards/4pplet/ibe60/readme.md:1]: Thanks! Fixed!

File: keyboards/4pplet/ibe60/matrix_diagram.md
[the remark this answers is not in the document]
Comment (4pplet) [reply keyboards/4pplet/ibe60/matrix_diagram.md:1]: Yes, that is correct, I have updated this.

**Comment by 4pplet** [comment]
Thanks for feedback! All should be fixed now

File: keyboards/4pplet/ibe60/matrix_diagram.md
[the remark this answers is not in the document]
Comment (4pplet) [reply keyboards/4pplet/ibe60/matrix_diagram.md:1]: Let's remove it if not needed 👍

**Review by drashna — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**